src/utils/api.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+const ncNews = axios.create({
+  baseURL: "https://nc-news-2sb7.onrender.com/api/",
+});
+
+export const fetchArticles = () => {
+  return ncNews.get(`/articles`);
```

Comment:
if you accessed the response here in a .then and narrowed down what you return, you wouldn't need to destructure inside the useEffect